### PR TITLE
Content: MOAR Marauder Collectibles

### PR DESCRIPTION
**Pull request by LocalGod79**
still working on the ship specs and bounty missions, but here’s a peek
at some of what should fill the gap between the 0.8.11 bounty missions
and the Marauder variants introduced in #831

more to come…soon…

### CODE REVIEW

**Comment by LocalGod79** [comment]
@Wrzlprnft for those images, again!

**Comment by LocalGod79** [comment]
many of the ships still require a new outfit loadout, but this is where we are now, if you have any feedback.  if not, we'll finish up in a day or two.

**Comment by jmhorjus** [comment]
"under the patchwork that expect has made an already deadly" should be "under the patchwork that _you_ expect has made an already deadly".

Other than that this looks awesome at first glance. Can't wait till I get some time free to try it.

**Comment by LocalGod79** [comment]
@jmhorjus - thanks for the report - fixed, will sync tonight.

the submitted ships have not been properly outfitted, yet - so there's nothing to really try (except perhaps job distribution, on landing) - the Bounder images came in yesterday, but i got otherwise occupied.  need to resize and gun/engine/turret port map.

outfits are progressing - ships purchased, loaded out, now: copy/paste/format.  also follow-up will be a change to the ships's price points - the upgraded outfits push the value of some of the hulls to negative credits.

@endless-sky i noticed you removed the Laser Rifles from the merged Marauder ships - i understand this move on Heavy Warships - should i include them in the smaller ships (to increase likelihood of death benefits)?  up to what tier?  i.e. Light Warship, Medium Warship...

**Comment by LocalGod79** [comment]
i believe that makes it ready to play

**Comment by jmhorjus** [comment]
+		word
 +			"one of "
 +			"any of "
 +			"some of "
 +		word
 +			"modified "
 +			"Marauder "

Should be "one of those ", "any of those ", "some of those ".

File: data/persons.txt [hunk 1]
```diff
@@ -127,3 +127,88 @@ person "Cap'n Pester"
...
+			"Splinters "
+			"Mantas "
+			"Arrows "
+			"Bounders "
+			"Firebirds "
+		word
+			"lately?"
+			"recently?"
+			"in the past couple of days?"
+#not sure about implying there is a shop that people can have this done to their ships, but it’s fun banter…
```
Comment (Wrzlprnft) [inline data/persons.txt]: make it something like "our forum thread" or "our site" or "our contact data"

**Comment by Wrzlprnft** [comment]
oops

**Comment by Wrzlprnft** [comment]
Make another word construct, and add
"our"
- "shop"
-"site"
-"hoard of ideas"
-"birthplace of ideas"
-"forum thread"
-"local god of coding" (dunno how much sense this makes but it fits in there)

**Comment by LocalGod79** [comment]
i've been over the changed files a few times - this looks as ready to merge as i am able to make it.

@endless-sky would you prefer the fleets to be in the fleets.txt, so they can be called upon in other situations, as well?

**Comment by jmhorjus** [comment]
After playing with them for only a short time I can say I really love these ships. 
Awesome job @Wrzlprnft @LocalGod79 !

**Comment by LocalGod79** [comment]
in light of ( [a48ac0a](https://github.com/endless-sky/endless-sky/commit/a48ac0a20dd35904327765fae81734f7e797c55b) ), upper tier ships have been returned to variant status (since the entire ship was already defined - great work maintaining backward compatability, @endless-sky ).

Marauder fleets are in 'fleets.txt' so they can be called upon in other missions and systems as Pirates using "Marauder fleet I/II/III/IV/V...IX/X", and jobs.txt redefines their government/personality upon calling them.

otherwise, i'm a little unsure if some of the bumps i've given to the smaller warships were excessive.  Jan's endorsement is both heartening (everyone likes to get approval) and concerning (people generally like unbalanced stuff.)

**Comment by LocalGod79** [comment]
i think i might need to redo some of the ships - looks like i messed up the leviathans, possibly also the falcons....considering closing this pull and creating a new one

**Comment by LocalGod79** [comment]
#864 has been merged!

**Comment by LocalGod79** [comment]
funny thing about the descriptions: there's no way for the user to see them unless they hack the game and make them available in a shipyard...

**Comment by jmhorjus** [comment]
Yeah I was looking for a way to view those descriptions and couldn't find one.  Really, they should be shown in the shipyard in your ship's details section, where it currently lists none of your ships stats or details at all except the price calculation.

**Comment by LocalGod79** [comment]
#869

**Pull request closed without merging**